Method reference fails to parse if more than one argument

If I add a second argument to a method reference like the following

```
-   :  { xidLength > 0 }? fixedArray[xidLength]
+   :  { xidLength > 0 }? fixedArray[xidLength, arg2]
```

Then the parser fails with the following error:

```
[ERROR] Failed to execute goal org.infinispan.maven-plugins:protocol-parser-generator-maven-plugin:1.0.6.Final:generate (default) on project infinispan-server-hotrod: Error in file /tmp/infinispan/server/hotrod/src/main/resources/hotrod.gr: hotrod.gr:389: Expected ']', found EOF! -> [Help 1]
```

--- a/protocol-parser-generator/src/main/java/org/infinispan/ppg/generator/Parser.java
+++ b/protocol-parser-generator/src/main/java/org/infinispan/ppg/generator/Parser.java
@@ -420,6 +420,7 @@ public class Parser {
                      params.add(new Action(ctx.ns, tokens, ctx.file, ctx.line));
                      tokens = new ArrayList<>();
                   } else tokens.add(token);
+                  break;
                case "[":
                   ++level;
                   // no break;

